client/my-sites/media-library/media-file.tsx

```diff
@@ -55,10 +54,9 @@ const MediaFile: React.FC< MediaFileProps > = function MediaFile( {
 export default connect( ( state: IAppState, { src }: Pick< MediaFileProps, 'src' > ) => {
 	const siteId = getSelectedSiteId( state );
 	const siteSlug = getSelectedSiteSlug( state ) as string;
-	const isAtomic = !! isSiteAutomatedTransfer( state, siteId as number );
-	const isPrivate = !! isPrivateSite( state, siteId ?? 0 );
 	const { filePath, query, isRelativeToSiteRoot } = mediaURLToProxyConfig( src, siteSlug );
-	const useProxy = Boolean( isAtomic && isPrivate && filePath && isRelativeToSiteRoot );
+	const isJetpack = isJetpackSite( state, siteId ?? 0, { treatAtomicAsJetpackSite: false } );
```

Comment:
Instead of doing `isJetpackSite` for `siteId` of `0`, I'd do (keep in mind also the rename I suggested above): 

```suggestion
	const isJetpackNotAtomic = siteId && isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } );
```